Gruntfile.js

```diff
@@ -77,7 +77,7 @@ module.exports = function(grunt) {
           reporter: 'spec',
           timeout: 15000
         },
-        src: 'nmclicktests/*.js'
+        src: 'nmclicktests/spec.commands.js'
```

Comment:
It was a mistake by me that slipped through.  It will be fixed in next round of nm clicktests coming in.  😭